test/xcode-specs.js

```diff
@@ -64,4 +64,12 @@ describe('xcode @skip-linux', () => {
     path.slice(-suffix.length).should.equal(suffix);
   });
 
+  it('should get max iOS SDK version', async() => {
+    let version = await xcode.getMaxIOSSDK();
+
+    should.exist(version);
+    (typeof version).should.equal('string');
+    (version-"6.1").should.be.at.least(0);
```

Comment:
you'll want to `parseFloat` these; sometimes versions can be like "6.0.1", which would result in NaN